### PR TITLE
fix: fix the condition when arc-focus-inside with no focusable element inside

### PR DESCRIPTION
**Pull request by xiaoxli**

### CODE REVIEW

File: src/focus.service.ts [hunk 1]
```diff
@@ -574,6 +574,11 @@ export class FocusService {
     if (directional && ev.next !== null) {
       this.selectNode(ev.next, scrollSpeed);
       return true;
+    } else if (directional && ev.next === null) {
+        if (this.selected) {
+          this.selected.dispatchEvent(new Event('arcselectingundefined', { bubbles: true, cancelable: true }));
```
Comment (connor4312) [inline src/focus.service.ts]: Should not be cancelable.

**Comment by connor4312** [comment]
Make sure to add this to the docs in the readme too.

File: src/focus.service.ts (hunk 1 above)
Comment (xiaoxli) [reply src/focus.service.ts]: Added

**Comment by xiaoxli** [comment]
This PR is not needed anymore

**Pull request closed without merging**